account/src/swagger.json

```diff
@@ -38,16 +38,62 @@
                             "application/json": {
                                 "schema": {
                                     "type": "object",
```

Comment:
I think you should remove the type here, because you already have type on schema